### DISCUSSION TITLE
GHI to POA Transposition without DNI/DHI

### DISCUSSION THREAD

**Discussion by oscarmenz** [pvlib/pvlib-python#1410] [Q&A]
The GHI to POA Transposition example [here](https://pvlib-python.readthedocs.io/en/stable/auto_examples/plot_ghi_transposition.html) requires DNI and DHI - not common measurements taken in the field. 

Suppose I have a dataset of only GHI data - how could I use this function to convert to POA given only these points, facility location, and time of year?

**Comment by kandersolar** [comment]
Hi @oscarmenz, what you need is a decomposition/separation model that estimates DNI and DHI from GHI.  Then you can use those component estimates with a transposition model to get POA.  pvlib has several decomposition models; see here: https://pvlib-python.readthedocs.io/en/stable/api.html#dni-estimation-models

Edit: in the latest docs, this is the correct link: https://pvlib-python.readthedocs.io/en/stable/reference/irradiance.html#dni-estimation-models

**Comment by cwhanse** [comment]
You can use one of the decomposition models in pvlib/irradiance.py, e.g., Erbs, to get DHI and DNI from GHI. There are several choices, see documentation [here](https://pvlib-python.readthedocs.io/en/stable/api.html#dni-estimation-models).
